subtests/docker_cli/run_cgroup_parent/run_cgroup_parent.py

```diff
@@ -110,6 +110,10 @@ def postprocess(self):
                     raise DockerTestFail("cgroup line does not conform to"
                                          " '<n>:<controller>:<path>': '%s'"
                                          % line)
+                # bz1385924: 'pids' fails in docker-1.10; not worth fixing.
+                if m.group(2) == 'pids':
```

Comment:
If one controller was added, another could be as well (net, user, etc.).  What's your thought on having a ``controller_skip_csv=pids`` config. option?  Not necessary ATM to be sure, but down the line it could save somebody else a bunch of time.